<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph2dScatteredWindowControlWidget</class>
 <widget class="QWidget" name="Graph2dScatteredWindowControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>41</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="dataSourceButton">
     <property name="text">
      <string>&amp;Data Source</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="axisButton">
     <property name="text">
      <string>Axis &amp;Setting</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="drawButton">
     <property name="text">
      <string>Draw &amp;Setting</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fontButton">
     <property name="text">
      <string>&amp;Font Setting</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
